Libraries/LibJS/Runtime/Object.cpp

```diff
@@ -107,6 +108,69 @@ Value Object::get_own_property(const Object& this_object, const FlyString& prope
     return value_here;
 }
 
+Value Object::get_own_properties(const Object& this_object, GetOwnPropertiesMode kind, bool enumerable_only) const
```

Comment:
Wait who calls this? I don't see a caller

Reply:
Oops, looks like that sneaked in from #2014. I decided to close that and open two separate PRs, and this code belongs in the other one